pymongoexplain/__main__.py

```diff
@@ -48,6 +49,23 @@ def new_func(self: Collection, *args, **kwargs):
     setattr(Collection, old_func_name, make_func(old_func, old_func_name))
 
 if __name__ == '__main__':
-    for file in sys.argv[1:]:
-        with open(file) as f:
-            exec(f.read())
+    parser = argparse.ArgumentParser(description="This script is intended to "
+                                                 "provide an easy method to "
+                                                 "run explain on all commands "
+                                                 "in a script. Find more "
+                                                 "documentation at "
+                                                 "https://github.com/mongodb-labs/pymongoexplain/")
+    parser.add_argument(
+        "input_script", nargs=1,help="The script that you "
+                                                      "wish to run explain on.")
+    parser.add_argument(
+        "arguments", metavar="script_arguments", help="add arguments to "
+                                                      "explained script"
+                                                        , nargs="?")
+
+    args = parser.parse_args()
+    file = args.input_script[0]
+    with open(file) as f:
```

Comment:
Can we add a test script to `test/` and run it in one job in Travis to smoke-test this functionality?

Reply:
Done.